python/setup.py

```diff
@@ -10,7 +10,7 @@
 
 setup(
     name='http_ece',
-    version='0.6.3',
+    version='0.6.5',
```

Comment:
Again, I'd urge that this be a significant change, since it breaks Draft 06 compliant libraries.